libr/bin/pdb/pdb_downloader.c

```diff
@@ -18,150 +18,110 @@ static bool checkExtract() {
 	return true;
 }
 
-static bool checkCurl() {
-	const char nul[] = R_SYS_DEVNULL;
-	if (r_sys_cmdf ("curl --version > %s", nul) != 0) {
+static bool download_and_write(SPDBDownloaderOpt *opt, const char *file) {
+	char *dir = r_str_newf ("%s%s%s%s%s",
```

Comment:
do `dbg->file` and `guid` come from the user or from the binary? If from the latter, they should be assumed untrusted so we should escape them.

Reply:
see https://github.com/radareorg/radare2/pull/16966#discussion_r435585946